bin/docker-to-squash.py

```diff
@@ -0,0 +1,1430 @@
+                      default=3, help="Replication factor for all files uploaded to HDFS")
+  parser.add_argument("--hadoop-prefix", type=str, dest='hadoop_prefix',
+                      default=os.environ.get('HADOOP_PREFIX'),
+                      help="hadoop prefix value for environment")
+  parser.add_argument("-f", "--force", dest='force',
+                      action="store_true", default=False, help="Force overwrites in HDFS")
+  parser.add_argument("--check-magic-file", dest='check_magic_file',
+                      action="store_true", default=False, help="Check for a specific magic file"
+                      + "in the image before uploading")
+  parser.add_argument("--magic-file", type=str, dest='magic_file',
+                      default='etc/dockerfile-version', help="The magic file to check for"
+                      + "in the image")
+  parser.add_argument("--max-layers", type=int, dest='MAX_IMAGE_LAYERS',
+                      default=37, help="Maximum number of layers an image is allowed to have")
+  parser.add_argument("--max-size", type=int, dest='MAX_IMAGE_SIZE',
+                      default=10*1024*1024*1024, help="Maximum size an image is allowed to be")
+  parser.add_argument("-b", "--bootstrap", dest='bootstrap',
+                      action="store_true", default=False, help="Bootstrap setup"
+                      + " of required HDFS directories")
+  return parser
+
+def check_dependencies():
+  global HADOOP_BIN_DIR
+  try:
+    command = [HADOOP_BIN_DIR + "/hadoop", "version"]
+    shell_command(command, False, False, True)
+  except:
+    logging.error("Could not find hadoop. Make sure HADOOP_PREFIX " +
+                  "is set correctly either in your environment or on the command line " +
+                  "via --hadoop-prefix")
+    return 1
+
+  try:
+    command = ["skopeo", "-v"]
+    shell_command(command, False, False, True)
+  except:
+    logging.error("Could not find skopeo. Make sure it is installed and present " +
+                  "on the PATH")
+    return 1
+
+  try:
+    command = ["/usr/sbin/mksquashfs", "-version"]
+    shell_command(command, False, False, True)
+  except:
+    logging.error("Could not find /usr/sbin/mksquashfs. Make sure squashfs-tools is installed " +
+                  "and /usr/sbin/mksquashfs is present on the the PATH")
+    return 1
+
+  try:
+    command = ["tar", "--version"]
+    shell_command(command, False, False, True)
+  except:
+    logging.error("Could not find tar. Make sure it is installed and present " +
+                  "on the PATH")
+    return 1
+
+  try:
+    command = ["setfattr", "--version"]
+    shell_command(command, False, False, True)
+  except:
+    logging.error("Could not find setfattr . Make sure it is installed and present " +
+                  "on the PATH")
+    return 1
+
+  return 0
+
+def main():
+  global LOG_LEVEL
+  global HADOOP_PREFIX
+  global HADOOP_BIN_DIR
+  global MAX_IMAGE_LAYERS
+  global MAX_IMAGE_SIZE
```

Comment:
Global var MAX_IMAGE_SIZE is not defined in this module

Reply:
They are passed in from arguments